src/main/java/bio/terra/pdao/gcs/GcsPdao.java

```diff
@@ -150,4 +152,36 @@ public void deleteFilesFromStudy(Study study) {
             blob.delete();
         }
     }
+
+    private enum AclOp {
+        ACL_OP_CREATE,
+        ACL_OP_DELETE
+    };
+
+    public void setAclOnFiles(String studyId, List<String> fileIds, String readersPolicyEmail) {
+        fileAclOp(AclOp.ACL_OP_CREATE, studyId, fileIds, readersPolicyEmail);
+    }
+
+    public void removeAclOnFiles(String studyId, List<String> fileIds, String readersPolicyEmail) {
+        fileAclOp(AclOp.ACL_OP_DELETE, studyId, fileIds, readersPolicyEmail);
+    }
```

Comment:
this is currently not needed. i'm not sure if we will ever need this. i'm wondering if we should remove this and simplify the code below for now. and then add back the delete when/if needed. the only case i can think of where it might be needed is soft delete.